zulipterminal/model.py

```diff
@@ -179,24 +179,40 @@ def get_message_ids_in_current_narrow(self) -> Set[int]:
             current_ids = self.index['all_starred']
         return current_ids.copy()
 
-    @asynch
-    def _start_presence_updates(self) -> None:
+    def _start_presence_initially(self) -> None:
```

Comment:
Do you have a reason for including this method? It's only called once and the 2 calls can just go inline in the `__init__` if needed?

Reply:
Yes we can add it in `__init__` since it's asynchronous.

Reply:
I'm not sure precisely what you meant by asynchronous here, but it seems to be a good refactor :+1: